cavefish-server/cavefish/tests/test/Spec.hs

```diff
@@ -6,34 +6,30 @@
 module Spec (spec) where
```

Comment:
Generates build warnings:
- `Defined but not used`
-  import does not have explicit import list